src/UI/Launcher.jsx

```diff
@@ -30,6 +31,9 @@ class Launcher extends Component {
 	}
 }
 
-Launcher.propTypes = {onClick: PropTypes.func};
+Launcher.propTypes = {
+	messengerState: PropTypes.string,
```

Comment:
_pseudo code_
```suggestion
	messengerState: PropTypes.oneOf(['open', 'minimize']).isRequired,
```

of nog beter als je die string waardes kan ophalen van een centrale plek zodat we ze maar 1x hoeven op te schrijven

Reply:
Ik heb dit aangepast in een commit 4d2d099b4387e1fb3a4f132545923e54278dc8e9